src/main.ts

```diff
@@ -475,3 +475,8 @@ export type {
 export * from './types';
 
 export { doCommandFromClient, promisify } from './utils';
+
+export {
+  RECONNECTED,
+  DISCONNECTED,
+} from './events';
```

Comment:
(commenting here for threading)

@njooma it would be a bigger change, but we could define it as an enum. I think that would be the most clear for usage and naming for export. We could consider adding `connecting`, `connected`, etc here as well.

```
export enum MachineConnectionEvent {
  RECONNECTED = 'reconnected',
  DISCONNECTED = 'disconnected',
}
```

Reply:
Ya agreed, I think we should add it now. Also would it really be a big change? Seems like we just add your `enum` to `events.ts` and export that, right? We don't actually have to change anything else since the string values are still the same

Reply:
I would want to start using the enum internally as well - not crazy but a bigger diff.

Reply:
Ya for sure, but also the SDK team could take care of that if needed. Probably is easier/faster for you to do it now, but seemed out of scope

Reply:
Quickly made this change, let me know what you think.